server/README.md

```diff
@@ -0,0 +1,131 @@
+
+### Kubernetes (kind)
+In this section we'll launch the the same multi-container application we used above but this time we'll use [kind](https://kind.sigs.k8s.io/) to spin up a local Kubernetes cluster and `kubectl` to launch the deployment.
+
+#### Prerequisites
+Ensure both `kind` and `kubectl` are installed locally.
+
+* [Installing kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+* [Installing kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+#### Build and launch the sample deployment
+From the [samples](./samples) directory run `docker build` to build the docker image for the provided Python client:
+
+```console
+[user@machine samples]$ docker build -t sample-client clients/python
+Sending build context to Docker daemon  3.375MB
+Step 1/15 : FROM python:3.7-alpine as base
+... snipped
+Step 14/15 : ENTRYPOINT ["python", "appencryption_client.py"]
+ ---> Running in a0958391c52e
+Removing intermediate container a0958391c52e
+ ---> 838466cf27e6
+Step 15/15 : CMD ["--help"]
+ ---> Running in e94395d01e20
+Removing intermediate container e94395d01e20
+ ---> c0a177a10252
+Successfully built c0a177a10252
+Successfully tagged sample-client:latest
+```
+
+we'll also need the server image:
+
+```console
+[user@machine samples]$ docker build -t asherah-server ../go
+Sending build context to Docker daemon  87.04kB
+Step 1/19 : ARG GOVERSION=1.13
+... snipped
+Step 18/19 : ENTRYPOINT ["/asherah-server"]
+ ---> Using cache
+ ---> 2fdfbe426e0e
+Step 19/19 : CMD ["--help"]
+ ---> Using cache
+ ---> a3bc3c0483a8
+Successfully built a3bc3c0483a8
+Successfully tagged asherah-server:latest
+```
+
+Now that we have both of the images we can run `kind` to create our local Kubernetes cluster and then load the new images into the cluster:
+
+```console
+[user@machine samples]$ kind create cluster
+...
+[user@machine samples]$ kind load docker-image sample-client
+...
+[user@machine samples]$ kind load docker-image asherah-server
+...
+```
+
+And now we're ready to use `kubectl` launch the deployment defined in [deployment.yaml](./samples/deployment.yaml):
+
+```console
+[user@machine samples]$ kubectl --context kind-kind apply -f deployment.yaml
+deployment.apps/myapp created
+```
+
+We can view our pod and the application logs to verify everything is in working order:
+
+```console
+[user@machine samples]$ kubectl --context kind-kind get po
+NAME                     READY   STATUS    RESTARTS   AGE
+myapp-77bd786698-ls42l   2/2     Running   0          70s
+[user@machine samples]$ kubectl --context kind-kind logs -c myapp myapp-77bd786698-ls42l | head -n2
+INFO:root:starting test
+INFO:root:starting session for partitionid-1
+```
+
+And when finished, `kind delete cluster` can be used to delete the cluster.
+
+## Server Development
+
+### Code generation
+First you'll need to ensure the protobuf compiler is installed...
```

Comment:
Similar to the above links in the readme, can also provide a link/instruction how to install protobuf compiler